arangod/Agency/AgencyFeature.cpp

```diff
@@ -63,8 +63,8 @@ AgencyFeature::AgencyFeature(application_features::ApplicationServer& server)
       _supervisionTouched(false),
       _waitForSync(true),
       _supervisionFrequency(1.0),
-      _compactionStepSize(1000),
-      _compactionKeepSize(50000),
+      _compactionStepSize(100),
```

Comment:
does the modification of these parameters have anything to do with the bugfix or is unrelated?

Reply:
@fceller Can give details, but it is related to a different but related problem. Imo this could be a different PR against 3.6 or?